### PR TITLE
On 1748 new user request access

### PR DESCRIPTION
**Pull request by arshedmust**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Views/OrganisationSearch/Results.cshtml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     bool showErrors = TempData["ShowErrors"] as bool? ?? false;
 }
 
-<a asp-action="Index" asp-controller="OrganisationSearch" class="govuk-back-link">Search again</a>
+@*<a asp-action="Index" asp-controller="OrganisationSearch" class="govuk-back-link">Search again</a>*@
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/OrganisationSearch/Results.cshtml]: Did you mean to comment this out? - if it won't be brought back soon it should be removed.

Comment (arshedmust) [reply src/SFA.DAS.AssessorService.Web/Views/OrganisationSearch/Results.cshtml]: Yes I meant to comment this out. Will see when testers test it if this is still needed. According to the story it has been removed. But user cannot now go back to the original page.

Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Web/Views/OrganisationSearch/Results.cshtml]: You need to query that with a BA and get the AC's to clarify and then remove it or put it back in

Comment (alandburns) [reply src/SFA.DAS.AssessorService.Web/Views/OrganisationSearch/Results.cshtml]: Let's get this to the testers.

**Comment by alandburns** [comment]
Has been merged to ON-1637

**Pull request closed without merging**